<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="inputFilePathLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Input files path:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inputFileEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tempTextBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>351</y>
      <width>256</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="runOnceButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Run once</string>
    </property>
   </widget>
   <widget class="QLabel" name="ifDeleteInFileLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete input file?</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="ifDeleteInFileBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>49</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="deleteInFileButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="notDeleteInFileButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>No</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="outputFileLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Output file path:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputFileEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ifChangeOutFileLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>71</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Output file:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="ifChangeOutFileBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="overwriteOutFileButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Overwrite</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="modifyOutFileButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Modify</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="timerBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="multipleTimerLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Once per                       sec</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="timerSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="runByTimerButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>380</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Run by timer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>380</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop timer</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="timerCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Timer</string>
    </property>
   </widget>
   <widget class="QLabel" name="modificationLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Number for modification rule:</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="modificationBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <zorder>timerBox</zorder>
   <zorder>inputFilePathLabel</zorder>
   <zorder>inputFileEdit</zorder>
   <zorder>tempTextBrowser</zorder>
   <zorder>runOnceButton</zorder>
   <zorder>ifDeleteInFileLabel</zorder>
   <zorder>ifDeleteInFileBox</zorder>
   <zorder>outputFileLabel</zorder>
   <zorder>outputFileEdit</zorder>
   <zorder>ifChangeOutFileLabel</zorder>
   <zorder>ifChangeOutFileBox</zorder>
   <zorder>runByTimerButton</zorder>
   <zorder>stopButton</zorder>
   <zorder>timerCheckBox</zorder>
   <zorder>modificationLabel</zorder>
   <zorder>modificationBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFileModifier">
    <property name="title">
     <string>FileModifier</string>
    </property>
   </widget>
   <addaction name="menuFileModifier"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
